mountpoint-s3/src/inode.rs

```diff
@@ -1485,8 +1489,10 @@ impl InodeStat {
 pub enum InodeError {
     #[error("error from ObjectClient")]
     ClientError(#[source] anyhow::Error),
-    #[error("file does not exist")]
-    FileDoesNotExist,
+    #[error("file does not exist {0:?}")]
+    FileDoesNotExist(OsString),
+    #[error("directory does not exist at inode {0}")]
```

Comment:
See previous comment. But also the error message may be confusing: we know the given `ino` corresponds to the directory in the info, but are also saying that it does not exist.

Reply:
Yes, I should change it to only log Directory name if we distinguish this case from normal deleted inode case.
But, I am going ahead with removing this error case as you mentioned in the previous comment.